docs/installation.rst

```diff
@@ -2,62 +2,71 @@
+  \• ``pandeia>=1.7``: Exposure time calculator.
 
-	- Read and write FITS files.
-	- Read and write ASCII tables (specifically in the IPAC format).
-	- Generate Sersic profile models (if any are in the generated scene).
+  \• ``webbpsf>=1.0.0``: Nancy Grace Roman PSF calculator. STIPS also requires that ``poppy``, a
+  support package used by WebbPSF, have version ``>=1.0.3``.
 
-* `esutil`: Used for retrieving data from sqlite databases in the form of numpy arrays.
+  \• ``astropy``: STIPS uses Astropy in order to:
 
-.. warning::
-   `esutil` is not installed by the stips setup.py because its pip installation has
-   errors. After installing STIPS, you must run `pip install esutil --no-cache-dir` to
-   have `esutil` available.
+    \• Read and write FITS files.
 
-.. note::
-   `esutil` is only needed if you are using `stips.star_generator`
+    \• Read and write ASCII tables (specifically in the IPAC format).
+
+    \• Generate Sersic profile models (if any are in the generated scene).
+
+  \• ``montage_wrapper``: STIPS uses ``montage`` to generate mosaics. It is
+  only imported if STIPS is asked to generate a multi-detector image.
+
+  \• ``numpy``: STIPS uses ``numpy`` extensively for almost everything that it does.
 
-* `montage_wrapper`: STIPS uses montage to generate mosaics. It is only imported if
-  STIPS is asked to generate a multi-detector image.
-* `numpy`: STIPS uses numpy extensively for almost everything that it does.
-* `photutils`: STIPS uses photutils to determine the flux inside the half-light radius
+  \• ``photutils``: STIPS uses ``photutils`` to determine the flux inside the half-light radius
   in generated Sersic profiles.
-* `synphot>=1.1.1` and `stsynphot>=1.1.0`: STIPS uses synphot and stsynphot to generate
-  bandpasses, count rates, and zero points. Note that the reference data must
-  also be downloaded, as described below in :ref:`downloading-required-ref-data`.
-* `scipy`: STIPS uses scipy to manipulate its internal images (zoom and rotate).
 
-Finally, STIPS requires a set of data files whose location is marked by setting the 
-environment variable `stips_data`, which will be installed as part of these instructions.
+  \• ``synphot>=1.1.1`` and ``stsynphot>=1.1.0``: STIPS uses ``synphot`` and
+  ``stsynphot`` to generate bandpasses, count rates, and zero points. Note that
+  the reference data must also be downloaded, as described below in :ref:`downloading-required-ref-data`.
+
+  \• ``scipy``: STIPS uses SciPy to manipulate its internal images (zoom and rotate).
+
+  \• ``esutil``: Used for retrieving data from sqlite databases in the form of ``numpy`` arrays.
+
+.. warning::
+   ``esutil`` is not installed by the STIPS ``setup.py`` file because its pip
+   installation has errors. After installing STIPS, you must run ``pip install esutil --no-cache-dir``
+   to have ``esutil`` available.
+
+.. note::
+   ``esutil`` is only needed if you are using ``stips.star_generator``.
```

Comment:
I had to move `esutil` to the end of the requirements list because the warning and note blocks prevented the other packages' lines from rendering properly while using the bullet character.